bot/exts/evergreen/movie.py

```diff
@@ -142,9 +141,9 @@ async def get_pages(self, client: ClientSession, movies: Dict[str, Any], amount:
 
     async def get_movie(self, client: ClientSession, movie: int) -> Dict:
         """Get Movie by movie ID from TMDB. Return result dictionary."""
-        url = BASE_URL + f"movie/{movie}?" + urlencode(MOVIE_PARAMS)
+        url = BASE_URL + f"movie/{movie}"
```

Comment:
I think you should add a check that `movie` is actually an integer, either here or after line 135 where it's fetched, but not validated